site2/docs/security-jwt.md

```diff
@@ -236,6 +236,9 @@ tokenSecretKey=file:///path/to/secret.key
 
 # If using public/private (Note: key files must be DER-encoded)
 # tokenPublicKey=file:///path/to/public.key
+# Role names that are treated as "super-user", meaning they will be able to do all admin
+# operations and publish/consume from all topics
+superUserRoles=test-user
```

Comment:
This does not seem to be the necessary parameter to set up the authentication for the broker. This parameter is part of the authorization section and we already mentioned it in the authorization section: https://pulsar.apache.org/docs/en/security-authorization/#enable-authorization-and-assign-superusers

Reply:
if we don't set this parameter ,we can't grand `produce `permission to  `test-user`; in other words ,if  the role that we create token is not  the superUser;we can't produce,consume; because  we can not execute following statement;
`$ bin/pulsar-admin namespaces grant-permission public/default \
            --role test-user \
            --actions produce,consume
`

there is a excption
> WARN  org.apache.pulsar.client.admin.internal.BaseResource - [http://localhost:8080/admin/v2/namespaces/public/default/permissions/test-user] Failed to perform http post request: javax.ws.rs.NotAuthorizedException: HTTP 401 Unauthorized
HTTP 401 Unauthorized
Reason: HTTP 401 Unauthorized

Reply:
Hi @RobertIndie any thoughts on the @sakurafly123's reply? Thanks

Reply:
Good catch, both are correct! 

We need to split the authentication and authorization config, @momo-jun should be working on this.